### ISSUE TITLE
Add option to output to specified file or stdout

### ISSUE THREAD

**Issue by bcoles** [si9int/Acamar#3]
plz

**Comment by si9int** [comment]
I will add the option as argument  `-o` on relative path

**Comment by bcoles** [comment]
Thanks.

Personally, I would prefer if it defaulted to stdout (optionally disabled with `-q` quiet mode) with `-o` as an optional argument for output to file.

This would also allow removing the dodgy `system` call to `cat`.

**Comment by si9int** [comment]
yep, makes sense. I will re-code some parts, it's really a bit sloppy tho ;-)

**Comment by pesertakursustm** [comment]
how to save JSON Format?
